extensions/statsd-emitter/src/main/java/io/druid/emitter/statsd/StatsDEmitterConfig.java

```diff
@@ -0,0 +1,100 @@
+package io.druid.emitter.statsd;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.Preconditions;
+
+import java.util.Map;
+
+/**
+ */
+public class StatsDEmitterConfig
+{
+
+  public static enum MetricType {
+    count,
+    timer,
+    guage
+  }
+
+  @JsonProperty
+  final private String hostname;
+  @JsonProperty
+  final private Integer port;
+  @JsonProperty
+  final private String prefix;
+  @JsonProperty
+  final private Map<String, MetricType> metricTypes;
+
+  @JsonCreator
+  public StatsDEmitterConfig(
+      @JsonProperty("hostname") String hostname,
+      @JsonProperty("port") Integer port,
+      @JsonProperty("prefix") String prefix,
+      @JsonProperty("metricTypes") Map<String, MetricType> metricTypes)
+  {
+    Preconditions.checkNotNull(hostname, "StatsD hostname cannot be null.");
+    Preconditions.checkNotNull(port, "StatsD port cannot be null.");
+    Preconditions.checkNotNull(metricTypes, "Must provide StatsD Metric Type mapping.");
+    Preconditions.checkArgument(!metricTypes.isEmpty(), "Must map at least 1 StatsD Metric Type");
```

Comment:
i think it will be painful to provide such map via properties file.
First please provide a default map that has all the mapping, generally users are not expert so they need a default template.
Second please make this a path and read the map from the file.